static/app/components/actions/archive.tsx

```diff
@@ -136,7 +142,19 @@ function ArchiveActions({
             disabled={disabled}
           />
         )}
-        menuTitle={t('Archive')}
+        menuTitle={
+          <MenuWrapper>
```

Comment:
@vuluongj20 This is the best way to do this yeah?

Reply:
Yeah…